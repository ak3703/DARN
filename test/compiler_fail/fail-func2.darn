void bar(int a, bool b, int a) {} /* duplicate formal a in bar */

int main() {
	return 0;
}
