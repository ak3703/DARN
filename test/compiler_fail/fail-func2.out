Fatal error: exception Failure("Duplicate formal a in bar")
